tests/Query/Filter/PathField.test.ts

```diff
@@ -12,6 +12,77 @@ function testTaskFilterForTaskWithPath(
     testFilter(filter, builder.path(path), expected);
 }
 
+declare global {
+    namespace jest {
+        interface Matchers<R> {
+            toMatchTaskWithPath(path: string): R;
+            toBeValid(): R;
```

Comment:
It's interesting to me that the Path tests use `toBeValid` but the Description ones don't.

If you found them to be not needed in Description, should they be deleted from Path tests, to simplify those?

Reply:
@claremacrae The absence of the validity test was, frankly, due to me only modifying the existing test to use marchers. Validity could be just as useful. I opted not to go that far due to time constraints.

Reply:
This isn't worth worrying about. Resolving conversation.